### ISSUE TITLE
Widget clicks customizations

### ISSUE THREAD

**Issue by smnthermes** [MarkaPola/Linkificator#16]
Is this API able to restore widget clicks customizations? https://bugzilla.mozilla.org/show_bug.cgi?id=1405031

**Comment by MarkaPola** [comment]
Yes, with these new capabilities, I can enhance mouse clicks handling.
